### PR TITLE
Update Installation/README

### PR DESCRIPTION
**Pull request by MaelRL**
## Summary of Changes

There were inconsistencies with what was actually being shipped.

Also, switch it to markdown and clean it up here and there.

## Release Management

* Affected package(s): `Installation`
* Issue(s) solved (if any): -
* Feature/Small Feature (if any): n/a
* License and copyright ownership: n/a

### CODE REVIEW

File: Installation/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- `CHANGES.md`: history of changes of CGAL
+- `CGALConfig.cmake`:
+- `CGALConfigVersion.cmake`:
+- `CMakeLists.txt`: main CMake project configuration file
+- `INSTALL.md`: information about the installation process
+- `LICENSE`: describes the license of CGAL
+- `LICENSE.BSL`: text of the Boost Software License (BSL) license
+- `LICENSE.COMMERCIAL`: text of the GeometryFactory Commercial license
+- `LICENSE.GPL`: text of the GNU General Public License (GPL) v3 license
+- `LICENSE.LGPL`: text of the GNU LESSER GENERAL PUBLIC License (LGPL) v3 license
```
Comment (albert-github) [inline Installation/README.md:16]: LESSER GENERAL PUBLIC -> Lesser General Public

File: Installation/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+README
+======
+
+This distribution of CGAL includes:
+
+- `AUTHORS`: current and former authors of CGAL
+- `CHANGES.md`: history of changes of CGAL
+- `CGALConfig.cmake`:
+- `CGALConfigVersion.cmake`:
```
Comment (albert-github) [inline Installation/README.md:9]: To be done?

File: Installation/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- `LICENSE.GPL`: text of the GNU General Public License (GPL) v3 license
+- `LICENSE.LGPL`: text of the GNU LESSER GENERAL PUBLIC License (LGPL) v3 license
+- `LICENSE.RFL`: text of the ETH Zurich Random Forest algorithm License (RFL) license
+- `README.md`: this file
+- `VERSION`: version number of this release of CGAL
+- `auxiliary/`: directory containing miscellaneous things, such as icons needed for the demos on macOS
+- `cmake/`: some scripts needed by the CMake build system
+- `data/`: directory containing various geometric data used by examples and demos of CGAL
+- `demo/`: many demo programs for the various packages of CGAL. These programs produce visual output.
+- `doc_html/`: documentation, e.g. the CGAL Installation Guide
```
Comment (albert-github) [inline Installation/README.md:24]: e.g. -> a.o.

File: Installation/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- `examples/`: programs illustrating the various packages of CGAL. These programs do not produce visual output.
```
Comment (albert-github) [inline Installation/README.md:25]:
```
programs illustrating the various packages of CGAL. These programs do not produce visual output.
```
->
```
programs illustrating the usage of the various packages of CGAL. These programs do not produce any visual output.
```

File: Installation/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- `include/`: the include files for the CGAL library (as well as small external libraries, see the file `LICENSE`)
+- `lib/`:
```
Comment (albert-github) [inline Installation/README.md:27]: To be completed?

**Review by lrineau — APPROVED** [review]

**Review by albert-github — APPROVED** [review]

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-147

**Pull request merged**